tests/dags/test_on_kill.py

```diff
@@ -25,6 +25,11 @@
 
 class DummyWithOnKill(DummyOperator):
     def execute(self, context):
+        import os
```

Comment:
@nuclearpinguin Did you fix these issue in other PR?

Reply:
This wasn't an issue, I was just making doubley sure that the tests created more processes.